pkg/controller/controller_utils.go

```diff
@@ -55,6 +62,12 @@ const (
 	ExpectationsTimeout = 5 * time.Minute
 )
 
+var UpdateTaintBackoff = wait.Backoff{
```

Comment:
Yes we should (as it's really, really bad right now), but it's better to wait until move to field is done, so we won't need to duplicate the work.

@aveshagarwal